modules/administration/pages/tls-certs-monitoring.adoc

```diff
@@ -0,0 +1,88 @@
+[[tls-certs-monitoring]]
+= TLS Certificates for Monitoring
+
+By default, {productname} does not provide any certificates for securing monitoring configuration.
+For providing security, you can import custom certificates, self-signed or signed by a third party certificate authority (CA).
+
+This section demonstrates how to generate client/server certificates for Prometheus and Node exporter minions self-signed with SUSE Manager CA.
+
+The host name of the TLS keys and certificates must match the fully qualified host name of the machine you deploy them on.
+You can set the host names in the ``X509v3 Subject Alternative Name`` section of the certificate.
+You can also list multiple host names if your environment requires it.
+
+
+== Generate client/server keys and certificates for monitoring
+
+To generate TLS keys and certificates for use with Prometheus server and node exporters you can follow these instructions:
+
+.Procedure: Create TLS keys and certificates
+
+. On the {productname} Server, change to the ``ssl-build`` directory:
++
+----
+cd /root/ssl-build
+----
+. Create private key:
++
+----
+openssl genrsa -out server.key 2048
+----
+. Create Certificate Signing Request (CSR) using the private Key.
+  An example OpenSSL configuration can be found below.
+  Please ensure that the set-cname parameter is the fully qualified domain name of your SUSE Manager Server.
++
+----
+openssl req -sha256 -text -config rhn-minion-openssl.cnf
+    -new -key server.key -out server.csr
+----
+. Create and sign certificate with CA:
++
+----
+openssl ca -extensions req_server_x509_extensions -outdir ./
+    -config rhn-ca-openssl.cnf -in server.csr -batch
+    -cert RHN-ORG-TRUSTED-SSL-CERT -keyfile RHN-ORG-PRIVATE-SSL-KEY
+    -days 365 -md sha256 -out server.crt
+----
```

Comment:
Uhh, you suggest to sign it with the official CA? I think this is ok, but in this case I would drop this whole chapter and handle every monitored client like a proxy and use the existing tooling we have.

In this case you can also use /root/ssl-build as directory.
You also do not need to describe how the ssl config should look like.

Reply:
You are right. For some reason my first tests with this tool hadn't worked for me.
Now retested and everything works as a charm. I think that's the best approach.
I'll clean up.

Reply:
Added shorter section about certificates generation to monitoring page. Thanks.